compiler/rustc_incremental/src/persist/dirty_clean.rs

```diff
@@ -139,7 +139,7 @@ pub fn check_dirty_clean_annotations(tcx: TyCtxt<'_>) {
         return;
     }
 
-    // can't add `#[rustc_clean]` etc without opting in to this feature
+    // can't add `#[rustc_clean]` etc without opting into this feature
```

Comment:
`opt*in*to` is complicated, there are definitely strong differences of opinion on it.

Reply:
i dont have an opinion here, applying the change is less work for us both